ast/Trees.cc

```diff
@@ -448,18 +448,14 @@ string MethodDef::showRaw(const core::GlobalState &gs, int tabs) {
     buf << nodeName() << "{" << '\n';
     printTabs(buf, tabs + 1);
 
-    buf << "flags =";
```

Comment:
if we were to add a flag, this won't print it.
WDYT about storing `this->flags` in a local and resetting bits in it as you print them with an ENFORCE that no bits should be set after?

Reply:
I see these flags as no different from any other field in a struct. Every custom printing thing we have in Sorbet suffers from the same problem: if you add a field, nothing forces you to also update the printing code.

While I think the bit resetting thing you're suggesting is clever, I'm trying to actively move away from treating flags as bits, and towards them as their own fields.